plugins/woocommerce/tests/performance/requests/merchant/orders-search.js

```diff
@@ -0,0 +1,48 @@
+import { sleep, check, group } from "k6";
+import http from "k6/http";
+import { Trend } from "k6/metrics";
+import { randomIntBetween } from "https://jslib.k6.io/k6-utils/1.1.0/index.js";
+import { base_url, think_time_min, think_time_max } from "../../config.js";
```

Comment:
`product_search_term` needs to be included in this import

Reply:
Thanks, that was missing. Added it now, please just confirm so I could merge it after.